src/components/HeartbeatBar.vue

```diff
@@ -273,31 +284,40 @@ export default {
 }
 
 .hp-bar-big {
-    .beat {
+    .beat-hover-area {
         display: inline-block;
-        background-color: $primary;
-        border-radius: $border-radius;
 
-        &.empty {
-            background-color: aliceblue;
+        &:not(.empty):hover {
+            transition: all ease-in-out 0.15s;
+            opacity: 0.8;
+            transform: scale(var(--hover-scale));
         }
 
-        &.down {
-            background-color: $danger;
-        }
+        .beat {
+            background-color: $primary;
+            border-radius: $border-radius;
 
-        &.pending {
-            background-color: $warning;
-        }
+                        /*
+                        pointer-events needs to be changed because
+                        tooltip momentarily disappears when crossing between .beat-hover-area and .beat
+                        */
```

Comment:
fixed more linting

```suggestion
            /*
            pointer-events needs to be changed because
            tooltip momentarily disappears when crossing between .beat-hover-area and .beat
            */
```